src/scripts.ts

```diff
@@ -47,14 +46,17 @@ export class ScriptLinter extends EventEmitter {
   };
 
   private lint = (files: string[]) => {
-    const lintOptions = {configuration: this.configuration, formatter: "verbose"};
+    const lintOptions: ILinterOptions = {fix: false, formatter: "verbose"};
 
     return Promise.all(files.map(file =>
       utils.readFile(file).then(content => ({file, content}))))
       .then((fileObjs) =>
         new Promise((resolve, reject) => {
           for (let fileObj of fileObjs) {
-            const lintingResult = new Linter(fileObj.file, fileObj.content, lintOptions).lint();
+            const linter = new Linter(lintOptions);
```

Comment:
Now that we don't have to bind the linter to one file with the constructor, we could instantiate it only once, right?